styles/styles.css

```
@@ -114,24 +108,22 @@ body {
   margin: 0;
   padding: 0;
   display: flex;
-  justify-content: space-between;
   flex-wrap: wrap;
 }
 
 .advantages-item {
-  width: 400px;
+  width: 33%;
```

```suggestion
  width: 33.33%;
```

А то один процент останется, а от 1200 пикселей это будет заметно